python/py-Pillow/Portfile

```diff
@@ -52,10 +53,12 @@ if {${name} ne ${subport}} {
                         port:openjpeg \
                         port:freetype
 
-    patchfiles          patch-setup.py.diff
+    if {[lsearch {26 33} ${python.version}] == -1} {
```

Comment:
Looks like setup.py in 3.4.2 [1] should be patched, too. It has almost the same statements as patched-out ones in 5.2.0. Specifically, it looks into /opt/local/{lib,include}, so the build can fail if a user installs MacPorts to a non-default prefix.

[1] https://github.com/python-pillow/Pillow/blob/3.4.2/setup.py

Reply:
sorry, my bad... you're certainly right. I'll add another patch file so that it will work. Going about removing these subports from all other ports will require quite some time, which I don't have right now.